### PR TITLE
fixes #1822 - PHP 7.3 Warnings

### LINKED ISSUE #1822: PHP 7.3 Warnings
Running the unit tests with PHP 7.3.0 RC4 fails:

```
$ /tmp/php-7.3.0RC4/sapi/cli/php vendor/phpunit/phpunit/phpunit
PHPUnit 4.8.36 by Sebastian Bergmann and contributors.

EEE.E....

Time: 38 ms, Memory: 8.00MB

There were 4 errors:

1) Dompdf\Tests\AutoloaderTest::testAutoload
"continue" targeting switch is equivalent to "break". Did you mean to use "continue 2"?

/tmp/dompdf/src/Dompdf.php:610
/tmp/dompdf/src/Autoloader.php:38
/tmp/dompdf/src/Autoloader.php:38
/tmp/dompdf/tests/Dompdf/Tests/AutoloaderTest.php:17

2) Dompdf\Tests\DompdfTest::testConstructor
"continue" targeting switch is equivalent to "break". Did you mean to use "continue 2"?

/tmp/dompdf/lib/Cpdf.php:467
/tmp/dompdf/src/Adapter/CPDF.php:198
/tmp/dompdf/src/CanvasFactory.php:57
/tmp/dompdf/src/Dompdf.php:294
/tmp/dompdf/tests/Dompdf/Tests/DompdfTest.php:15

3) Dompdf\Tests\DompdfTest::testSetters
"continue" targeting switch is equivalent to "break". Did you mean to use "continue 2"?

/tmp/dompdf/src/Css/Stylesheet.php:1364
/tmp/dompdf/src/Dompdf.php:296
/tmp/dompdf/tests/Dompdf/Tests/DompdfTest.php:31

4) Dompdf\Tests\DompdfTest::testRender
"continue" targeting switch is equivalent to "break". Did you mean to use "continue 2"?

/tmp/dompdf/src/Renderer/Text.php:141
/tmp/dompdf/src/Renderer.php:263
/tmp/dompdf/src/Renderer.php:106
/tmp/dompdf/src/Renderer.php:194
/tmp/dompdf/src/Renderer.php:194
/tmp/dompdf/src/FrameReflower/Page.php:148
/tmp/dompdf/src/FrameDecorator/AbstractFrameDecorator.php:894
/tmp/dompdf/src/Dompdf.php:848
/tmp/dompdf/tests/Dompdf/Tests/DompdfTest.php:65

FAILURES!
Tests: 9, Assertions: 56, Errors: 4.
```

While using DomPDF, I found a few more Warnings in other places:

```
Warning: "continue" targeting switch is equivalent to "break". Did you mean to use "continue 2"? in /tmp/dompdf/src/Dompdf.php on line 631

Warning: "continue" targeting switch is equivalent to "break". Did you mean to use "continue 2"? in /tmp/dompdf/lib/Cpdf.php on line 474

Warning: "continue" targeting switch is equivalent to "break". Did you mean to use "continue 2"? in /tmp/dompdf/lib/Cpdf.php on line 481

Warning: "continue" targeting switch is equivalent to "break". Did you mean to use "continue 2"? in /tmp/dompdf/lib/Cpdf.php on line 488
```
I guess there are some more problems like these, which are not covered by unit tests yet.

PHP 7.3 will be released in 6 weeks (December 6, 2018). It improves performance of DomPDF a lot, see https://github.com/dompdf/dompdf/issues/1813#issuecomment-433648227

Would be nice if DomPDF is compatible from day 1.

### PR DESCRIPTION
**Pull request by PHPGangsta**

### CODE REVIEW

File: src/Css/Stylesheet.php [hunk 1]
```diff
@@ -1361,7 +1361,7 @@ private function _parse_css($str)
                                 $key = $page_selector;
 
                             default:
-                                continue;
+                                break;
```
Comment (bsweeney) [inline src/Css/Stylesheet.php]: I think we should maybe consider that the previous incarnation of the code was written incorrectly. If we go based on the common understanding of `continue` (short circuit to the next iteration of the looping structure) then dropping out of the switch may not have been the intended operation. Even though [based on the PHP documentation](http://php.net/manual/en/control-structures.continue.php) we were indeed just `break`ing out of the switch statement.

Perhaps we should go with:

```suggestion
                                continue 2;
```

File: src/Renderer/Text.php [hunk 2]
```diff
@@ -138,7 +138,7 @@ function render(Frame $frame)
 
             switch ($text_deco) {
                 default:
-                    continue;
+                    break;
```
Comment (bsweeney) [inline src/Renderer/Text.php]: Original intention may have been:

```suggestion
                    continue 2;
```

?

**Comment by bsweeney** [comment]
I've not run with PHP 7.3 yet, thanks for taking a look at these issues.

**Comment by PHPGangsta** [comment]
I didn't want to change the code flow, so I replaced the `continue` with `break`, because in that situation `continue` works like a `break` (see the link you posted).

I don't know what the original intention was, the safest way to fix the Warnings should be `break`. If your changes don't break existing working code, then apply your suggested fixes. I just want to get the Warnings fixed, otherwise the error.log fills the hard drive very quickly 😄  (and writing Warnings/Notices to the error.log costs performance and uses extra I/O).

If you want to test dompdf with PHP 7.3:

```
cd /tmp
wget https://downloads.php.net/~cmb/php-7.3.0RC5.tar.gz
tar -xzvf php-7.3.0RC5.tar.gz
cd php-7.3.0RC5/
./configure --prefix=/usr/local/php7.3.0RC5 --with-zlib --with-config-file-path=/usr/local/php7.3.0RC5/etc --enable-mbstring --with-gd
make

If you want to run the php tests (and report all failing tests):
make test

/tmp/php-7.3.0RC5/sapi/cli/php -v
cd /tmp/dompdf
/tmp/php-7.3.0RC5/sapi/cli/php /tmp/dompdf/vendor/phpunit/phpunit/phpunit
```

**Comment by bsweeney** [comment]
I've been using phpbrew for testing against multiple versions of PHP. It's just been acting a bit wonky lately so I haven't yet added 7.3. Plus I don't have as much time to put into this as I used to.

I understand the reason for the changes you made. I was only making suggestions based on reading the code to understand the original intention. But if you'd rather I made those changes that's fine.

**Pull request merged**